src/Lykke.Job.TxDetector/Sagas/ConfirmationsSaga.cs

```diff
@@ -65,10 +81,8 @@ public class ConfirmationsSaga
 
         private async Task Handle(BitcoinCashInRegisteredEvent evt, ICommandSender sender)
         {
-            await _log.WriteInfoAsync(nameof(ConfirmationsSaga), nameof(BitcoinCashInRegisteredEvent), evt.ToJson());
-
-            ChaosKitty.Meow();
-
+            _log.WriteInfo(nameof(BitcoinCashInRegisteredEvent), evt, "");
```

Comment:
maybe add some Msg ... "Handle received message". It's can help for found all log by handle